src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.component.html

```diff
@@ -3,85 +3,48 @@
-                </dt>
-                <dd>
-                    <input
-                        formControlName="datasetName"
-                        type="text"
-                        name="new_name"
-                        class="form-control"
-                        id="rename-field"
-                        data-test-id="rename-dataset-input"
-                        (keyup)="changeName()"
-                        [class.error-border-color]="renameError$ | async"
-                    />
-                </dd>
-            </dl>
-            <button
-                class="ms-4 rename-btn"
-                type="button"
-                (click)="renameDataset()"
-                [disabled]="!renameDatasetForm.valid"
-                data-test-id="rename-dataset-button"
-            >
-                Rename
-            </button>
-            <div *ngIf="datasetNameControl.invalid && (datasetNameControl.touched || datasetNameControl.dirty)">
-                <span
-                    *ngIf="datasetNameControl.errors?.required"
-                    class="text-danger fs-12"
-                    data-test-id="rename-dataset-error-name-required"
-                    >Name is required</span
-                >
-                <span
-                    *ngIf="datasetNameControl.errors?.pattern"
-                    class="text-danger fs-12"
-                    data-test-id="rename-dataset-error-name-pattern"
-                    >Invalid dataset name</span
-                >
-            </div>
-            <div *ngIf="renameError$ | async as renameError">
-                <div class="error-block" data-test-id="rename-dataset-error-custom">{{ renameError }}</div>
-            </div>
-        </form>
-        <div class="mt-4">
-            <h2>Danger Zone</h2>
-            <div class="danger-zone__content d-flex justify-content-between align-items-center w-75 mt-2">
-                <div class="danger-zone__description">
-                    <p class="description">Delete this dataset</p>
-                    <p class="hint mt-2">Once you delete a dataset, there is no going back. Please be certain.</p>
-                </div>
-                <button
-                    type="button"
-                    [disabled]="isDeleteDatasetDisabled"
-                    (click)="deleteDataset()"
-                    data-test-id="delete-dataset-button"
-                >
-                    Delete this dataset
-                </button>
-            </div>
-        </div>
+        <ng-container *ngIf="activeTab === settingsTabsEnum.GENERAL">
+            <app-general [datasetBasics]="datasetBasics" [datasetPermissions]="datasetPermissions"></app-general>
+        </ng-container>
+        <ng-container *ngIf="isSchedulingAvailable && activeTab === settingsTabsEnum.SCHEDULING">
+            <app-scheduling [datasetBasics]="datasetBasics" [datasetPermissions]="datasetPermissions"></app-scheduling>
```

Comment:
These settings are dataset specific. The names are too general. Esp., `app-general`.

Reply:
Fixed